### PR TITLE
Add student stuff, change User/Student/College

### PR DESCRIPTION
**Pull request by ENoll78**
Try #2.  inheritance and stuff.

### CODE REVIEW

File: src/main/java/com/collegecompendium/backend/models/College.java [hunk 1]
```diff
@@ -37,4 +31,8 @@ public class College extends User {
 	@Min(100)
 	@NotNull
 	private Integer cost;
+	
+	public List<Degree> getDegrees() {
```
Comment (meta1203) [inline src/main/java/com/collegecompendium/backend/models/College.java:41]: What is the reasoning behind this? Does Lombok not create the getter correctly for the degrees field>

File: src/main/java/com/collegecompendium/backend/models/User.java [hunk 2]
```diff
@@ -95,7 +95,10 @@ public void setLocation(String location) {
 	public String getId() {
 		return id;
 	}
-
+	public void setId(String val) {
```
Comment (meta1203) [inline src/main/java/com/collegecompendium/backend/models/User.java:92]: This doesn't correctly set the ID.

**Review by meta1203 — CHANGES_REQUESTED** [review]
The majority of this looks really good, just a couple of small bugs I caught. Fix them and I'll go ahead and merge.

**Review by Mwaume — APPROVED** [review]
Approving as requested

File: src/main/java/com/collegecompendium/backend/models/College.java (hunk 1 above)
Comment (ENoll78) [reply src/main/java/com/collegecompendium/backend/models/College.java:41]: Lombok's getters from @Data did not resolve this issue, had to input it manually.

File: src/main/java/com/collegecompendium/backend/models/User.java (hunk 2 above)
Comment (ENoll78) [reply src/main/java/com/collegecompendium/backend/models/User.java:92]: Fixed!

**Review by ENoll78 — COMMENTED** [review]
Looks good with changes to how User is being handled now.

**Review by meta1203 — APPROVED** [review]
Looks good

**Pull request merged**